apis/grpc/v1/payload/payload.pb.go

```diff
@@ -818,6 +818,8 @@ type Search_Config struct {
 	IngressFilters *Filter_Config `protobuf:"bytes,6,opt,name=ingress_filters,json=ingressFilters,proto3" json:"ingress_filters,omitempty"`
 	// Egress filter configurations.
 	EgressFilters *Filter_Config `protobuf:"bytes,7,opt,name=egress_filters,json=egressFilters,proto3" json:"egress_filters,omitempty"`
+	// Minimum number of result to be returned.
+	MinNum uint32 `protobuf:"varint,8,opt,name=min_num,json=minNum,proto3" json:"min_num,omitempty"`
```

Comment:
I suggest 'MinResult' instead of 'MinNum' as the wording 'num' can mean different things
```suggestion
	MinResult uint32 `protobuf:"varint,8,opt,name=min_result,json=minResult,proto3" json:"min_result,omitempty"`
```

Reply:
I explained it verbally, but I'll leave the information in text as well, just in case.
Currently, when doing a search, the `num` field of `Config` is used to specify the maximum number of searches. Considering this, I think it would be better to use `num` for anything related to the number of searches. As a result, I finally choose `min_num`.